addons/website/views/website_templates.xml

```diff
@@ -178,6 +178,12 @@
     <xpath expr="//footer//span[hasclass('o_footer_copyright_name')]" position="replace">
         <span class="o_footer_copyright_name mr-2">Copyright &amp;copy; Company name</span>
     </xpath>
+    <!--
```

Comment:
Keep master diff out of the stable diff :)
You can just mention it in a comment/review on this PR, but don't add it into the community code :+1: